translations/default/AdminCatalogFeature.xlf

```diff
@@ -1645,51 +1609,134 @@
-        <note>Line: 274</note>
+        <note>Line: 275</note>
+      </trans-unit>
+      <trans-unit id="9bccc98ae31786b6062c540931d1467e">
+        <source>Reduction tax</source>
+        <target>Reduction tax</target>
+        <note>Line: 313</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Form/Admin/Product/ProductVirtual.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="58fd2b2308056ad80255a322b305742b">
+        <source>Number of days</source>
+        <target>Number of days</target>
+        <note>Line: 125</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Form/Admin/Product/ProductWarehouseCombination.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="7bf2d26eab899c413218b729d4d914b7">
+        <source>Stored</source>
+        <target>Stored</target>
+        <note>Line: 62</note>
+      </trans-unit>
+      <trans-unit id="ddedc546753b8ca7f0ecdad9d0863f95">
+        <source>Location (optional)</source>
+        <target>Location (optional)</target>
+        <note>Line: 69</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Form/Admin/Sell/Product/Category/CategoriesType.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="5deec35f9bfc92d125dfaf8dccc39308">
+        <source>Add categories</source>
+        <target>Add categories</target>
+        <note>Line: 87</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/BulkCombinationPriceType.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="7828a35d153636336475fa0774877bde">
+        <source>If the price per unit of this combination is different from the initial price per unit, enter the value of the impact (negative or positive).</source>
+        <target>If the price per unit of this combination is different from the initial price per unit, enter the value of the impact (negative or positive).</target>
+        <note>Line: 118</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/BulkCombinationStockType.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="515447b09cce54e693574f0c98fadc6e">
+        <source>Edit fixed quantity</source>
+        <target>Edit fixed quantity</target>
+        <note>Line: 89</note>
       </trans-unit>
-      <trans-unit id="9bccc98ae31786b6062c540931d1467e">
-        <source>Reduction tax</source>
-        <target>Reduction tax</target>
-        <note>Line: 312</note>
+      <trans-unit id="f0a309e6326af053d5d8839c9cb41c73">
+        <source>Receive a low stock alert by email</source>
```

Comment:
Duplicate of line 1043

Reply:
I checked and we had `Receive a low stock alert by email ` with a space in the end, which is why it's a doublon in the xlf file, fixed it in code base, xlf will be generated later

Reply:
Perfect then, thank you!